src/Components/PollList/PollList.jsx

```diff
@@ -49,70 +50,81 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-// function createData(
-//   poll_name,
-//   poll_question,
-//   start_date,
-//   end_date,
-//   participants,
-//   status,
-//   action
-// ) {
-//   return {
-//     poll_name,
-//     poll_question,
-//     start_date,
-//     end_date,
-//     participants,
-//     status,
-//     action,
-//   };
-// }
-
-const rows = [
-  
-];
-
 const PollList = () => {
   const classes = useStyles();
-  const [openLogOut, setOpenLogOut] = useState(false);
-  const [openDelete, setOpenDelete] = useState(false);
   const [offset, setOffset] = useState(0);
   const [polls, setPolls] = useState([]);
-  const [selectedID,setSelectedID] = useState();
   const [currentPage, setCurrentPage] = useState(1);
+  const [selectedID, setSelectedID] = useState();
   const [page, setPage] = useState(0);
   const navigate = useNavigate();
   const URL = `https://dev.oppi.live/api/admin/v1/polls?offset=${offset}&limit=10&direction=desc&search=`;
   const SIGNOUT_URL = "https://dev.oppi.live/api/admin/v1/auth/signout";
   const DEL_URL = "https://dev.oppi.live/api/admin/v1/polls";
-  
-  const [open, setOpen] = React.useState(false);
-  
-  const handleClickOpen = () => {
-    setOpen(true);
+
+  // delete function
+  const [open, setOpenDelete] = useState(false);
```

Comment:
tên này chung chung quá
```suggestion
  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
```